components/TransactionHistory.jsx

```diff
@@ -0,0 +1,50 @@
+let React = require('react');
+
+let {Table} = require('react-bootstrap');
```

Comment:
This syntax was used here for consistency with the other components also using react-bootstrap.